modules/test/playwright/pages/portal-language-override-web/LanguageOverridePage.ts

```diff
@@ -0,0 +1,135 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Locator, Page, expect} from '@playwright/test';
+
+import {clickAndExpectToBeVisible} from '../../utils/clickAndExpectToBeVisible';
+import fillAndClickOutside from '../../utils/fillAndClickOutside';
+import {PORTLET_URLS} from '../../utils/portletUrls';
+import {waitForSuccessAlert} from '../../utils/waitForSuccessAlert';
+
+export type TLanguageOverride = {
+	key: string;
+	languageIds: string[];
+	values: string[];
```

Comment:
Two arrays that relate by index for languageIds and values seems less than intuitive. How about just having an object for values there the languageId is the key and the translation is the value.
```
{
  key: 'my-language-key',
  values: {
    ['en-us']: 'My Language key'
  }
}
```

Reply:
Way clever, thanks